ui/lib/config-ui/addon/routes/login-settings/index.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRoute extends Route {}
+export default class LoginSettingsRoute extends Route {
+  @service api;
+  @service capabilities;
```

Comment:
I think we can remove this, too since the service is injected directly into the component.